src/agent.rs

```diff
@@ -983,6 +984,23 @@ where
 
 #[cfg(feature = "web_sys")]
 fn worker_new(name_of_resource: &str, is_module: bool) -> Worker {
```

Comment:
Should I return a `Result` here and handle all the `unwrap`s?

Reply:
Sure, but in a follow up PR preferably